From 02629fa02e96763db7460a930239cc93649a52f8 Mon Sep 17 00:00:00 2001
From: =?utf8?q?Elan=20Ruusam=C3=A4e?= <glen@delfi.ee>
Date: Thu, 22 Mar 2012 12:00:16 -0300
Subject: [PATCH] modprobe: fix typo in config dump: option->options

kmod uses "option MODNAME", not "options MODNAME"

This breaks scripts that depend in modprobe -c output grepping
---
 tools/kmod-modprobe.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/tools/kmod-modprobe.c b/tools/kmod-modprobe.c
index 4760682..f6d47dc 100644
--- a/tools/kmod-modprobe.c
+++ b/tools/kmod-modprobe.c
@@ -218,7 +218,7 @@ static int show_config(struct kmod_ctx *ctx)
 		{ "install", kmod_config_get_install_commands },
 		{ "remove", kmod_config_get_remove_commands },
 		{ "alias", kmod_config_get_aliases },
-		{ "option", kmod_config_get_options },
+		{ "options", kmod_config_get_options },
 		{ "softdep", kmod_config_get_softdeps },
 	};
 	size_t i;
-- 
1.7.6.5

